skfuzzy/fuzzymath/fuzzy_ops.py

```diff
@@ -661,6 +661,60 @@ def interp_membership(x, xmf, xx):
     return xxmf
 
 
+def interp_value(x, xmf, y):
+    """
+    Finds the degree of membership ``u(xx)`` for a given value of ``x = xx``.
+
+    Parameters
+    ----------
+    x : 1d array
+        Independent discrete variable vector.
+    xmf : 1d array
+        Fuzzy membership function for ``x``.  Same length as ``x``.
+    y : float
+        Membership function value at ``x``.
+
+    Returns
+    -------
+    xx : 1d array
+        Array of discrete singleton values on universe ``x`` whose
+        membership function value is y, ``u(xx[i])==y``.
+
+    Notes
+    -----
+    For use in Fuzzy Logic, where an interpolated discrete value ``xx`` which its
+    membership function value u(xx) equals ``y`` is given. Then, consider a
+    new value x = xx, which does not correspond to any discrete
+    values of ``x``. This function computes the value of ``x`` such that its
+    membership value ``u(xx)`` is ``y`` using linear interpolation.
+    """
+
+    indices = np.nonzero([True if (xmf[i]<=y<=xmf[i+1] or xmf[i]>=y>=xmf[i+1]) and (i==0 or xmf[i]!=y) else False for i in range(len(x)-1)])[0]
+
+    # We have len(indices) values in ``x``
+    xx = [0.0]*len(indices)
```

Comment:
If a user decided to input a value with no crossings, e.g., a membership function with a max of 0.5 but they pass 0.6 as `y`, `len(indices)` is zero and we return an empty list.

Do you think that's OK? I'd tend to lean towards yes, as it's technically correct (there were no possible answers) but we should document it so nobody gets surprised.